src/server/db/schema.ts

```diff
@@ -122,6 +128,18 @@ export const sexualOrientation = pgEnum("sexual_orientation", [
   "Other",
 ]);
 
+export const preregistrations = createTable("preregistration", {
```

Comment:
add a comment explaining this table, following the format of the other ones

Reply:
done